### ISSUE TITLE
Can't find variable "jasmine"

### ISSUE THREAD

**Issue by AvnerCohen** [jcarver989/phantom-jasmine#3]
Obviously I'm doing something wrong.. but I'd appreciate some help here as I'm just getting into phantomJS as well as Jasmine.

Working on Ubuntu 12.04, phantom js is working fine, I can get the various samples to run with no issues.

However once I run the bare bone example:
`phantomjs lib/run_jasmine_test.coffee examples/TestRunner.html`

I get:
`Can't open 'lib/run_jasmine_test.coffee'`

I tried to change:
`#!/usr/local/bin/phantomjs`

to:
`#!/usr/bin/phantomjs`

Since this is where my phantomJS is, but still the same issue.

For what it's worth, if I open TestRunner.html directly via browser, it works fine.

**Comment by jcarver989** [comment]
Hmm... have you tried passing the full path instead of the relative path to the run_jasmine_test.coffee ?

Eg. instead of phantomjs lib/run_jasmine_test.coffee try:

phantomjs /Users/joeblow/wherever-you-cloned-the-repo/lib/run_jasmine_test.coffee

**Comment by AvnerCohen** [comment]
Same thing `can't open...`

I can edit the file, so it's not some permission issue. 
The OS identifies it as a +x item, I tried to run it directly using (not sure if it makes any sense)
./run_jasmine_test.coffee
I get the exact same error ( `can't open...`)

For what it's worth, I also created a plain file with the header only:
`#!/usr/bin/phantomjs`

and than tried to run this via phantom js and I get the same error. So this doesn't seem to be any issue related directly to phantom-jasmine.

Is there any way to add verbose output to phantomjs to understand the reason for the "can't open" result ?
